core/App/utils/helpers.ts

```diff
@@ -28,6 +30,51 @@ export function hashToRGBA(i: number) {
   return '#' + '00000'.substring(0, 6 - colour.length) + colour
 }
 
+export function formatTime(time: Date, translate: TFunction, params?: { long?: boolean; format?: string }): string {
```

Comment:
Not sure if we should be passing the translate function as a parameter. Can we not just import it here directly and use it?

Reply:
No we can't because useTranslate() is a hook and hooks can't be used outside of components

Reply:
We should be able to export the i18n instance to access the translation function directly from the lang utility. Im not convinced that passing the translation function as a param is a good idea. It couples the formatter too closely to translation function.